README.md

```diff
@@ -218,6 +218,14 @@ Additional lists you might find useful:
 *Working with notification software.*
 
 - [Mercure plugin](https://github.com/josbeir/cakephp-mercure) - Push real-time updates to clients using the Mercure protocol.
+- [Cake/Notification plugin)(https://github.com/skie/notification) - The Notification plugin provides support for sending notifications across a variety of delivery channels.
```

Comment:
Same violation here, Cake is core prefix 
Should be `CakeNotification`

Also, you should only list the main plugin 
Rest needs to be communicated from its docs. Dependencies (package principles and func splitting) are not part of this here.